README.md

```diff
@@ -0,0 +1,38 @@
+# Swagger UI Field Formatter
+
+## Introduction
+
+This module provides a field formatter for file fields with allowed file types
+as json or yml, which renders the uploaded json or yml file using the Swagger UI
+if the uploaded json or yml file is a valid swagger file. This module uses the
+Swagger UI Library available at https://github.com/swagger-api/swagger-ui
+
+## Requirements
+
+This module does not have any special requirements. However in order to be able
+to allow json or yml file extensions in Drupal, you might need to install the
+file entity module.
```

Comment:
Just as an FYI, there is an update to the File Entity (fieldable files). The dev.developer site is running 7.x-2.12, but the recommended version is 7.x-2.16.